### ISSUE TITLE
YAML: Preserve one blank line for comments at the end of an object

### ISSUE THREAD

**Issue by noc7c9** [prettier/prettier#6445]
**Prettier 1.18.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEEBGArJAdKACPAYjzTgDMIAnOXA2ve40i63e+gazgE8k8A3AIYAbAK40o7fETyCy8SlLrTichSAA0qAA4wAltADOyUIMqUIAdwAKZhMZSD+EPQBNNINJUFguMAMraPnpQAObIMJTiWgAWMAC2wgDqMXrwhkFgcP72aXr8adzI4IbGWiGGcJQw1t6h8YLIZCKVWhiGAB4AQt6+cAGC8XAAMiFwTS1wbZ3+IaHCcACKohDwE8KtIEGUlZTF3IPCHtqUITBJbjAxyAAcAAxaJxCVSd7axSdwu-zjWgCOK3gtQg2gcIEEhgAtFA4HBXHCPNQAXpqLVBPVGkhmhspiBKvE9BEorjDHMFstVuMsZMtDBBGgLq4rsgAEy07x6YRzADCEHiDWKUGgvxAokqABV6Q5sZUAL6yoA)
```sh
--parser yaml
```

**Input:**
```yaml
obj:
  # before
  
  
  # before

  
  key: value

  
  # after
  
  
  # after
```

**Output:**
```yaml
obj:
  # before

  # before

  key: value
  # after
  # after

```

**Expected behavior:**
```yaml
obj:
  # before

  # before

  key: value

  # after

  # after

```
Specifically I expect it to match the behavior for Javascript.

**Comment by ikatyang** [comment]
Bug.

**Comment by fisker** [comment]
Fixed by #8356
